docs/UnitTestBarAndStandards.md

```diff
@@ -0,0 +1,23 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+     
+     ## Test requirements
+* All new non-UI code must have unit tests.
```

Comment:
>non-UI code [](start = 10, length = 11)

Since we have no UI, can we remove "non-UI code" ?

Reply:
```suggestion
* All new code must have unit tests.
```